test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -140,8 +152,9 @@ class test_uct_sockaddr : public uct_test {
 
 UCS_TEST_P(test_uct_sockaddr, connect_client_to_server)
 {
-    client->connect(0, *server, 0, m_connect_addr, NULL, NULL,
-                    &client->client_cb_arg);
+    client->connect(0, *server, 0, m_connect_addr, client_iface_priv_data_cb,
+                    NULL, NULL, &client->client_cb_arg);
```

Comment:
the field client_cb_arg should be renamed to max_conn_priv

Reply:
also, maybe pass 'this' as arg here as well?

Reply:
@yosefe you mean in order to have one pack_data_cb right?
in the next pr i'm working on, i'm changing 'this' for the cm case as the cb arg, so if it's not a major issue right now, i prefer to keep it as is. If something changes later, i'll unite these.
would this work?

Reply:
yes, if it's fixed in next PR it's also OK
what about renaming client_cb_arg to max_priv?

Reply:
it will be either fixed or changed so they won't be the same anymore and so there will be nothing to unite.

i'll do the renaming.